server/src/config/config.yaml.sample

```
@@ -11,6 +13,7 @@ typeorm:
   logging: true
 
 jwt:
-  secret:
-    access: 128bit_base64_encoded
-    refresh: 128bit_base64_encoded
+  secret: 128bit_base64_encoded
+
+auth:
+  rest_api: kakao_api
```

```suggestion
  kakao_client_id: kakao_api
```

oauth 규격에 맞게 client_id로 지정하는게 좋아보이지만 크게 상관은 없습니다.

---

변경했습니다!